### DISCUSSION TITLE
关于 CMCC RAX3000M 的 CPU

### DISCUSSION THREAD

**Discussion by cherinyy** [immortalwrt/immortalwrt#1070] [Q&A]
官方将其 CPU 标为 ZX279128R，也就是说原本在官方固件中，充当 CPU 的是 ZX279128R。
而在 ImmortalWrt 中，因为 MT7981 才有适配，所以把 MT7981 抓出来当 CPU，是吗？
那后续如果有 ZX279128R 的适配了，拿来当 CPU，对于设备来说是否会更好呢？

**Comment by 1715173329** [comment]
你买的版本是 RAX3000Z，不是 RAX3000M

**Reply by cherinyy** [reply]
> 你买的版本是 RAX3000Z，不是 RAX3000M

那就是官方标错了？

![Screenshot_2023-09-16-10-23-56-51_090d9ae461b065ca11c329abb9cd0d70](https://github.com/immortalwrt/immortalwrt/assets/9050713/bd282ad4-29cb-4e49-a729-97c524681104)

我买的是 RAX3000M。
